Marlin/Marlin_main.cpp

```diff
@@ -1464,7 +1464,7 @@ static void set_axis_is_at_home(const AxisEnum axis) {
     if (axis == Z_AXIS) {
       #if HOMING_Z_WITH_PROBE
 
-        current_position[Z_AXIS] -= zprobe_zoffset;
+        current_position[Z_AXIS] -= Z_PROBE_OFFSET_FROM_EXTRUDER;
```

Comment:
z_offset changes are already taken care of in the general case for all printers (see above)